common/scripts/lint-commits

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+die() {
+	printf '%s\n' "$*" >&2
+	exit 1
+}
+
+GIT_CMD=$(command -v chroot-git 2>/dev/null) ||
+GIT_CMD=$(command -v git 2>/dev/null) ||
+die "chroot-git and git couldn't be found"
```

Comment:
shouldn't it say `or` instead of `and`?

Reply:
Being a non-native Indo-European speaker, I don't know. I guess it's better to say "need either chroot-git or git"?